lms/djangoapps/courseware/module_render.py

```diff
@@ -518,6 +519,9 @@ def rebind_noauth_module_to_user(module, real_user):
     # to the Fragment content coming out of the xblocks that are about to be rendered.
     block_wrappers = []
 
+    if settings.FEATURES.get("LICENSING", False):
+        block_wrappers.append(wrap_with_license)
```

Comment:
That's not a bad idea. However, for context, the reason I implemented it this way (so that it would only show in the LMS) is because the Studio editor is, itself, an XBlock. When the license was showing up in Studio, it was also being rendered at the bottom of the Studio editor, which looked very strange.